### PR TITLE
Add and remove Branch

### PR DESCRIPTION
**Pull request by lazyCode-404**
### In this Branch I implemented;

- [ ] Remove all hardcoded items from the tasks array.

- [ ] Created a new JavaScript file for the new functionality called addremoveedit.js

- [ ] Add function for adding a new task (add a new element to the array).

- [ ] Add function for deleting a task (remove an element from the array).

- [ ] Add function for editing task descriptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
-# to-do-list
\ No newline at end of file
+# to-do-list
+
+>  To Do List Project For Microverse By Bwalya Nonde.
+
+
+> This is a simple web develoment project for microverse created using simple HTML(Hyper text mark-up   Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage. And JavaScript which is used for making the webpage responsive.
+
+![screenshot](to-do-list.png)
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly fix this, the image is not displayed and is broken.

File: src/addremoveedit.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const actualtodo = load();
+  const result = actualtodo.filter((activity) => activity.id === id);
+  element.parentNode.removeChild(element);
+  const e = theBigList.querySelectorAll('.list');
+  for (let i = 0; i < result.length; i += 1) {
+    result[i].index = i;
+    e[i].id = i;
+  }
+  save(result);
+}
```
Comment (sja-thedude) [inline src/addremoveedit.js]: - [ ] Kindly make sure that your items are added to the list and deleted when you click "clear all completed" button after checking it.
- [ ] Kindly also make sure that when you mark the check box there should be a line through the text.

_Right now the items are not added nor deleted. The below video is your app. kindly fix it_

https://user-images.githubusercontent.com/70269507/173155033-493fa8a4-7009-4058-b009-c7683a45d88e.mov

You can watch this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg&ab_channel=EmiliaAndrzejewska) to have a look at how items are added, edited and deleted.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @BwalyaNonde, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters are passing
✔️ Professional Readme
✔️ Good PR Title and Description

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/addremoveedit.js (hunk 2 above)
Comment (lazyCode-404) [reply src/addremoveedit.js]: @sja-thedude Hi thank you for your review and making me a better developer 💛 

Happy coding!!

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rustam-tolipov) [inline README.md:57]: - [ ] [OPTIONAL] Please don't forget to add MIT.md license file 📃

**Review by rustam-tolipov — APPROVED** [review]
## Hi @BwalyaNonde 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 3 above)
Comment (lazyCode-404) [reply README.md:57]: @Rustamxon7 Thank you for approving my project 👍.
Thanks for the reminder.

**Pull request merged**